SU2_CFD/src/solvers/CIncNSSolver.cpp

```diff
@@ -684,12 +684,9 @@ void CIncNSSolver::SetTau_Wall_WF(CGeometry *geometry, CSolver **solver_containe
 
       const su2double VelTangMod = GeometryToolbox::Norm(int(MAXNDIM), VelTang);
 
-      /*--- Compute normal distance of the interior point from the wall ---*/
+      /*--- Compute nearest-neighbor distance of the interior point from the wall ---*/
 
-      su2double WallDist[MAXNDIM] = {0.0};
-      GeometryToolbox::Distance(nDim, Coord, Coord_Normal, WallDist);
-
-      su2double WallDistMod = GeometryToolbox::Norm(int(MAXNDIM), WallDist);
+      su2double WallDistMod = geometry->vertex[iMarker][iVertex]->GetNearestNeighborDistance();
```

Comment:
revert

Reply:
Because the normal neighbor approach is being mixed with this and what we talked about was to replace where it was straightforward.